client/tests/e2e/permissions.js

```diff
@@ -54,7 +54,7 @@ test.serial("checking super user permissions", async t => {
 
   await validateUserCanEditUserForCurrentPage(t)
 
-  // User is super user, they may edit position of type super user for
+  // User is super-user, they may edit position of type super user for
```

Comment:
If you change "super user" to "super-user", at least be consistent on the rest of this line…